.gitlab-ci.yml

```
@@ -127,6 +131,16 @@ test:static:release:
   variables:
     BUILD_TYPE: Release
 
+trigger:mender-mcu-integration:
+  rules:
+  - if: $CI_COMMIT_BRANCH =~ /^(pr_[0-9]+)/
```

```suggestion
  - if: $CI_COMMIT_BRANCH =~ /^(pr_[0-9]+)$/
```

:point_up: if we want to be even more strict